docs/reference/garbage-collection.md

```diff
@@ -177,7 +177,7 @@ spark-submit --class io.treeverse.clients.GarbageCollector \
   -c spark.hadoop.lakefs.api.access_key=<LAKEFS_ACCESS_KEY> \
   -c spark.hadoop.lakefs.api.secret_key=<LAKEFS_SECRET_KEY> \
   -c spark.hadoop.fs.azure.account.key.<AZURE_STORAGE_ACCOUNT>.dfs.core.windows.net=<AZURE_STORAGE_ACCESS_KEY> \
-  s3://treeverse-clients-us-east/lakefs-spark-client-312-hadoop3/0.3.0/lakefs-spark-client-312-hadoop3-assembly-0.3.0.jar \
+  s3://treeverse-clients-us-east/lakefs-spark-client-312-hadoop3/0.5.0/lakefs-spark-client-312-hadoop3-assembly-0.5.0.jar \
```

Comment:
I know in the past we've had issues with anonymous access to these paths. @Jonathan-Rosenberg may have context on *why* (I certainly don't...). May be worth checking this?
certainly blocking for this PR!

Reply:
Thanks @arielshaqed!
> certainly blocking for this PR!

Blocking or non-blocking? :)

Reply:
Not :-)